### ISSUE TITLE
VideoWrapper not found in environment/wrappers.py

### ISSUE THREAD

**Issue by krishpop** [tensorflow/agents#64]
In the [Environments Colab tutorial](https://github.com/tensorflow/agents/blob/03f4a9e10b6a8b38427c8b6e87be9e72aaec3bce/tf_agents/colabs/2_environments_tutorial.ipynb) I saw a mention of the VideoWrapper class to save videos from the environment. However, it is currently not implemented in environment/wrappers.py. Is there any plan of releasing this wrapper?

**Comment by sguada** [comment]
Yeah we plan to release a VideoWrapper, we finishing up the dependencies for it.
